docs/wiki/source-files.md

```
@@ -1,6 +1,6 @@
 # Source Files
 
-[← Home](index.md)
+#home
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)